docs/translations/translations-in-FSE-templates.md

```diff
@@ -0,0 +1,121 @@
+	<div class="wp-block-woocommerce-empty-mini-cart-contents-block">
+		<!-- wp:paragraph {"align":"center"} -->
+		<p class="has-text-align-center">
+			<strong>Your cart is currently empty!</strong>
+		</p>
+		<!-- /wp:paragraph -->
+
+		<!-- wp:woocommerce/mini-cart-shopping-button-block -->
+		<div class="wp-block-woocommerce-mini-cart-shopping-button-block"></div>
+		<!-- /wp:woocommerce/mini-cart-shopping-button-block -->
+	</div>
+	<!-- /wp:woocommerce/empty-mini-cart-contents-block -->
+</div>
+<!-- /wp:woocommerce/mini-cart-contents -->
+```
+
+This FSE-template contains the following part:
+
+```html
+<p class="has-text-align-center">
+	<strong>Your cart is currently empty!</strong>
+</p>
+```
+
+Having this text hardcoded in a FSE-template causes two problems:
+
+1. This string can only be edited, when a user is using an FSE-theme, such as [Twenty Twenty-Two](https://wordpress.org/themes/twentytwentytwo/). If the user is using a non-FSE-theme, such as [Twenty Twenty-One](https://wordpress.org/themes/twentytwentyone/) or older, this FSE-template cannot be edited.
+2. Even is a user is using an FSE-theme, every user that is using a site language other than the default one, has to manually change the string `Your cart is currently empty!`.
+
+So, how can we use translations in FSE templates?
+
+First, we need to find the following code:
+
+```html
+<!-- wp:paragraph {"align":"center"} -->
+<p class="has-text-align-center">
+	<strong>Your cart is currently empty!</strong>
+</p>
+<!-- /wp:paragraph -->
+```
+
+We then replace this code with the following code:
+
+```html
+<!-- wp:pattern {"slug":"woocommerce/mini-cart-empty-cart-message"} /-->
+```
+
+In the file, that holds the logic for this FSE-template, in this case `src/BlockTypes/MiniCart.php`, we then register this placeholder as a block pattern:
+
+```php
+/**
+ * Register block pattern for Empty Cart Message to make it translatable.
+ */
+public function register_empty_cart_message_block_pattern() {
+    register_block_pattern(
+        'woocommerce/mini-cart-empty-cart-message',
+        array(
+            'title'    => __( 'Mini Cart Empty Cart Message', 'woo-gutenberg-products-block' ),
+            'inserter' => false,
+            'content'  => '<!-- wp:paragraph {"align":"center"} --><p class="has-text-align-center"><strong>' . __( 'Your cart is currently empty!', 'woo-gutenberg-products-block' ) . '</strong></p><!-- /wp:paragraph -->',
+        )
+    );
+}
+```
+
+Finally, we call this function using the following code:
+
+```php
+/**
+ * Initialize this block type.
+ *
+ * - Hook into WP lifecycle.
+ * - Register the block with WordPress.
+ */
+protected function initialize() {
+    parent::initialize();
+    add_action( 'wp_loaded', array( $this, 'register_empty_cart_message_block_pattern' ) );
+}
+```
+
+The PR for the implementation above can be found on https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6248/files.
```

Comment:
We should update the repo name for this and other links in this PR.

Reply:
Good catch. All URLs, within this PR, have now been replaced.